README.md

```diff
@@ -522,6 +522,7 @@ A curated list of awesome Java frameworks, libraries and software.
 *Tools for managing microservices, i.e. creation, coordination or discovery.*
 
 * [Apollo](https://spotify.github.io/apollo/) -  Libraries for writing composable microservices.
+* [Eureka](https://github.com/Netflix/eureka) -  REST based service registry for resilient load balancing and failover.
```

Comment:
Could you please delete the redundant whitespace at the beginning of the description. Bonus points for doing the same to the previous 'Apollo' entry.